### PR TITLE
fix(labels): 'making labels dynamic' 

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra <am1426620@gmail.com>
### Functionality:
to check and Make the labels dynamic everywhere

### Solution:
made the label calls dynamic everywhere and done some other changes 

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
step 1: check the labels everywhere in the codebase and
step 2: see in dashboard in S no -1 is showing as backlog.
step 3: also check task type label in task details

### CODE REVIEW

File: src/app/body/task-details/task-details.component.ts [hunk 1]
```diff
@@ -152,6 +152,7 @@ export class TaskDetailsComponent implements OnInit {
     })).subscribe({
       next: (data) => {
         this.task = data;
+        console.log(this.task);
```
Comment (sanjaykrishna1203) [inline src/app/body/task-details/task-details.component.ts:155]: Please remove the console logs

File: src/app/icons/generate-icon/generate-icon.component.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export class GenerateIconComponent implements OnInit {
 
   ngOnInit(): void {
     this.colorCode = "#" + this.icon.ColorCode;
+    console.log(this.icon);
```
Comment (sanjaykrishna1203) [inline src/app/icons/generate-icon/generate-icon.component.ts:19]: Please remove it here too

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

**Review by viv9k — APPROVED** [review]
Approved

File: src/app/body/task-details/task-details.component.ts (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/task-details/task-details.component.ts]: resolved

File: src/app/icons/generate-icon/generate-icon.component.ts (hunk 2 above)
Comment (abhishek-mishra019) [reply src/app/icons/generate-icon/generate-icon.component.ts]: resolved

**Pull request merged**